examples/http/crestron_tsw.json

```diff
@@ -0,0 +1,32 @@
+{
+    "logo": "crestron.svg",
+    "version": "1.0.0",
+    "name": "Crestron TSW",
+    "description": "This script its designed to monitor the device's status and synchronization with external calendars.",
```

Comment:
Collects information for crestron touch screen device's calendar syncronisations